pkg/util/export/batch_processor.go

```diff
@@ -402,20 +412,20 @@ func (c *MOCollector) doCollect(idx int) {
 	defer c.stopWait.Done()
 	ctx, span := trace.Start(c.ctx, "MOCollector.doCollect")
 	defer span.End()
-	logutil.Debugf("doCollect %dth: start", idx)
```

Comment:
What exactly is the difference of these two?

Reply:
diff is from here, L278, https://github.com/matrixorigin/matrixone/blob/main/pkg/util/export/batch_processor.go#L278

it marks all logs with `logutil.Discardable()` output by `c.logger`.
the mark `logutil.Discardable()` will trigger `DicardableCollect` logic while mo do collect.
https://github.com/matrixorigin/matrixone/blob/main/pkg/util/export/batch_processor.go#L349...L359